javascript/javascript1/week3/js1-week3.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Homework - JS1-week3</title>
+  </head>
+  <body>
+    <script src="item array removal.js"></script>
+    <script src="when will we be there.js"></script>
+    <script src="series duration.js"></script>
+    <script src="note app.js"></script>
```

Comment:
Check file names `itemArrayRemoval.js` etc

Reply:
Thanks.
I will use underscores (_) or dashes (-) next time.

Reply:
<img width="212" alt="Screenshot 2022-06-18 at 10 16 14" src="https://user-images.githubusercontent.com/10253713/174429234-e5a6c358-0c64-451a-ab67-5303d4d2b72a.png">
I am fine with files names (no need to use `-` or `_`). I want to say that you should write the file name as part of the src attribute. So instead of `<script src="item array removal.js"></script>`, based on the image above it should be `<script src="itemArrayRemoval.js"></script>`. Sorry if I wasn't clear.